src/guides/plugins/notebook.md

```diff
@@ -0,0 +1,464 @@
+# Notebook Plugin Development Guide
+
+_For general documentation on Plugins - click [here](/concepts/plugins/plugin_basics.md)._
+
+_For advanced documentation on Frontend Plugins - click [here](/concepts/plugins/frontend.md)._
+
+## What is Notebook?
+
+**Notebook** is a note taking plugin. The repository for Notebook can be found [here](https://github.com/ninjeeter/notebook).
```

Comment:
I want to make sure caido has access the code to any referenced example.
Lets transfer the ownership of the repository to https://github.com/caido-community, I added you as a maintainer of  that organization.

Reply:
Done.